lithium/lithium.py

```diff
@@ -268,7 +268,7 @@ def readTestcaseLine(line):
 def writeTestcase(filename):
     with open(filename, "w") as file:
         file.write(before)
-        for i in range(len(parts)):
+        for i in xrange(len(parts)):
```

Comment:
xrange() does not exist in python3 and the benefit is very likely unnoticeable. I'd stick with range.